apps/shell/tash_main.c

```diff
@@ -357,6 +368,9 @@ static int tash_main(int argc, char *argv[])
 
 		tash_free(line_buff);
 	} while (tash_running);
+#ifdef CONFIG_PM
+	tash_pm_close_driver();
+#endif
```

Comment:
why can't we do this inside the tash_stop function?

Reply:
We are not using the `pm_driver` after `while` loop. So, it's better to close it here only.

Reply:
yes but tash_running this cleared in the tash_stop function only.

Reply:
Yes, moved the `tash_pm_close_driver` inside `tash_stop`